examples/complex_sys/raze/remote/libgit2-sys-0.12.14+1.1.0.BUILD.bazel

```diff
@@ -0,0 +1,151 @@
+"""
+@generated
+cargo-raze crate build file.
+
+DO NOT EDIT! Replaced on runs of cargo-raze
+"""
+
+# buildifier: disable=load
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_test",
+)
+
+# buildifier: disable=load
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+package(default_visibility = [
+    # Public for visibility by "@raze__crate__version//" targets.
+    #
+    # Prefer access through "//complex_sys/raze", which limits external
+    # visibility to explicit Cargo.toml dependencies.
+    "//visibility:public",
+])
+
+licenses([
+    "notice",  # MIT from expression "MIT OR Apache-2.0"
+])
+
+# Generated targets
+# buildifier: disable=load-on-top
+load(
+    "@io_bazel_rules_rust//cargo:cargo_build_script.bzl",
+    "cargo_build_script",
+)
+
+# buildifier: leave-alone
+cargo_build_script(
+    name = "libgit2_sys_build_script",
+    srcs = glob(["**/*.rs"]),
+    crate_root = "build.rs",
+    edition = "2018",
+    deps = [
+        "@rules_rust_examples_complex_sys__cc__1_0_61//:cc",
+        "@rules_rust_examples_complex_sys__libssh2_sys__0_2_19//:libssh2_sys",
+        "@rules_rust_examples_complex_sys__libz_sys__1_1_2//:libz_sys",
+        "@rules_rust_examples_complex_sys__pkg_config__0_3_19//:pkg_config",
+    ] + selects.with_or({
+        # cfg(unix)
+        (
+            "@io_bazel_rules_rust//rust/platform:aarch64-apple-ios",
+            "@io_bazel_rules_rust//rust/platform:aarch64-linux-android",
+            "@io_bazel_rules_rust//rust/platform:aarch64-unknown-linux-gnu",
+            "@io_bazel_rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@io_bazel_rules_rust//rust/platform:i686-apple-darwin",
+            "@io_bazel_rules_rust//rust/platform:i686-linux-android",
+            "@io_bazel_rules_rust//rust/platform:i686-unknown-freebsd",
+            "@io_bazel_rules_rust//rust/platform:i686-unknown-linux-gnu",
+            "@io_bazel_rules_rust//rust/platform:powerpc-unknown-linux-gnu",
+            "@io_bazel_rules_rust//rust/platform:s390x-unknown-linux-gnu",
+            "@io_bazel_rules_rust//rust/platform:x86_64-apple-darwin",
+            "@io_bazel_rules_rust//rust/platform:x86_64-apple-ios",
+            "@io_bazel_rules_rust//rust/platform:x86_64-linux-android",
+            "@io_bazel_rules_rust//rust/platform:x86_64-unknown-freebsd",
+            "@io_bazel_rules_rust//rust/platform:x86_64-unknown-linux-gnu",
+        ): [
+            "@rules_rust_examples_complex_sys__openssl_sys__0_9_58//:openssl_sys",
+        ],
+        "//conditions:default": [],
+    }),
+    rustc_flags = [
+        "--cap-lints=allow",
+    ],
+    crate_features = [
+      "https",
+      "libssh2-sys",
+      "openssl-sys",
+      "ssh",
+      "ssh_key_from_memory",
+    ],
+    # LOCALLY MODIFIED
```

Comment:
For other reviewers: These files are slightly modified from the cargo-raze generated versions, because they require attributes raze doesn't currently set. This feels like a pretty reasonable approach, as if people re-gen the `git diff` will show the local modifications to re-apply (at least, until Raze supports them)?

Reply:
Is there an issue open on `cargo-raze` for what you ran into here?

Reply:
@UebelAndre I opened the following two issues:

- https://github.com/google/cargo-raze/issues/287
- https://github.com/google/cargo-raze/issues/288